### ISSUE TITLE
Subdomain not work?!

### ISSUE THREAD

**Issue by xstable** [Riant/host-switch-plus#62]
I have enterd a subdomain:
neu.ggshardt.de to IP: 94.130.152.89

![auswahl_481](https://user-images.githubusercontent.com/1778068/34203234-a1190ad8-e57a-11e7-9670-b8b0c0505e0f.jpg)

But If I enter this Domain, it complete the url to ggshardt.de/http//neu.ggshardt.de.
If I put neu.ggshardt.de in my /etc/hosts it works really well.

At the Page is only (an not still installed) wordpress-package. Maybe the redirect is the cause of some trouble?

**Comment by Riant** [comment]
Yes, the PAC_Script modal that this plugin used is different with /etc/hosts config, so for local Wordpress site, I do not know how to fix that.
